### DISCUSSION TITLE
魔改造Jeff

### DISCUSSION THREAD

**Discussion by christianprentice** [recursion-teamdev-green/project#15] [Ideas]
![BoscRArCAAAepo-](https://user-images.githubusercontent.com/51078652/185964168-41ded5e3-6c28-4364-a5d9-0a4e8f12faf8.jpg)
案１です、レビューお願いします

**Comment by dancing-amigo** [comment]
リスペクトもありつつ面白いです🤣

**Comment by christianprentice** [comment]
ガチクソコラです、すいません…
![dqnjeffsan](https://user-images.githubusercontent.com/51078652/186164980-e1432de3-472d-4475-93b2-610be7941d88.jpg)
![kausei](https://user-images.githubusercontent.com/51078652/186164988-26963477-7cfd-4a58-9398-aaff34f081b9.jpg)
![nicejeff](https://user-images.githubusercontent.com/51078652/186164993-2e92a0af-9b4d-4b3b-a059-7a203d38ea6c.jpg)

**Reply by christianprentice** [reply]
@takeshi8989 @jKai-Hw

**Reply by jKai-Hw** [reply]
爆笑しました🤣
一番最初の壁のやつはコメント思いついたんですが、他のはすぐにはちょっと思いつかないですね(笑)

**Reply by christianprentice** [reply]
クソコラってことで割り切って作りましたw

**Comment by jKai-Hw** [comment]
壁のやつのコメントは
「Recursionで学んでいくうちに、いろいろできるようになって調子乗ってないかい？そんな時は、Jeffryさんを思い出そう。ブラウザでフレームワークやらSQLやらを動くようにしてしますめちゃすごMetaエンジニアだ。そんなJeffryさんはCSの技術を自慢するのではなく、日本のソフトウェア界のためにその技術を用いている。Jeffryさんに感謝を伝えながら、学んだ知識は自分のためでなく人のために使おう！」
的な感じで(案の一つ、下書きのため表現もうちょい変えたい)、受け狙いだけでなく少し考えさせられるようなコメントどうでしょう？

**Comment by jKai-Hw** [comment]
目が光っているJeffryさんは
名前: 無敵Jeffry
コメント: 無敵状態のJeffryさん。無敵Jeffryを引き当てたあなたは、今後一週間、どんなエラーでも5秒で解決できる(かもしれない)能力を手にいれた！！！！
はどうでしょう。

金髪Jeffryさんは
名前: 王Jeffry
コメント: !!!!!warning!!!!! 王Jeffryと謁見中!!今すぐRecursionを使用している旨を伝え、周りがどんな状態でも大きな声で「ありがとうございます！」と画像に3回伝えましょう。そうすればJeffryさんがやさしく微笑んでいるように見えます。

チャリで来た盤
名前: チャリで来たJeffryさん
コメント: いつも優しい表情のJeffryさんですが、実は強面の中に入っても、あら不思議、すんなり馴染んじゃいます。皆さんもJeffryさんを見習って、どんな職場でも自分の力を遺憾なく発揮しましょう。

**Reply by christianprentice** [reply]
面白いと思います！
金髪Jefferyを見返してみると少しクオリティ低いので、３つに絞ってもいいのかなーと思ったのが感想です

**Comment by jKai-Hw** [comment]
↑一応ほかの候補も考えてみました。最後のチャリで来たはあんま関係ない人も一応写っているので、入れるかどうかは悩みどころですが、こんな感じでどうでしょうか？

**Comment by dancing-amigo** [comment]
いいですね！（笑）
Kaiさんの言う通りチャリで来たは抜いてもいいかもですね。
壁のコメントちょっと長いので、「Recursionユーザー全員の目標。ブラウザでフレームワークやらSQLやらを動くようにしているめちゃすごMetaエンジニアだ。いつまでも慢心せず、Jeffryさんのように誰かの役に立つソフトウェアを開発できるようこれからも精進しよう」とかどうでしょう？

**Reply by dancing-amigo** [reply]
URが三枚以上になるなら確率2%にしたほうがいいかもですね。じゃないと下手すると1000回とか引く羽目になるのでw

**Comment by jKai-Hw** [comment]
いいですね。確かに長いです、、、(笑)
なんかUR4枚だと、なんかレア感なくなる気がするので、3枚に絞ったほうが良い気がします。

**Comment by dancing-amigo** [comment]
正直三枚でも多いと思うんで二枚にしませんか？SRより多いのが違和感あるのと確率が渋すぎると思います

**Reply by christianprentice** [reply]
僕もそんくらいの塩梅がちょうどいい気がしますね！
どれにするかはお二人に任せたいです😂

**Reply by dancing-amigo** [reply]
自分的には壁は絶対入れたくて、その次は王Jeffですかね...
せっかく作ってもらってあれですが、UR１体のままでレア感を維持でもいいかなとも思います
そろそろ決めないとまずいですよね

**Reply by christianprentice** [reply]
壁は個人的に好きです、一番クオリティ高いですし。
他の画像は正直作ってみた程度のものなのであるどっちでも良いかと。

**Reply by dancing-amigo** [reply]
とりあえず壁だけいれて、他のは雰囲気みて追加していく形でいきましょう！
